cinema_service/settings.py

```diff
@@ -20,14 +20,17 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
+SECRET_KEY = os.environ.get(
+    "SECRET_KEY",
     "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
 )
```

Comment:
The `SECRET_KEY` should not have a hardcoded fallback value. Consider removing the default insecure key and ensure that the environment variable is set in all environments to enhance security.